README.md

```diff
@@ -381,6 +383,11 @@ Some helpful things for debugging:
 * Additionally set `NS1_DEBUG` environment variable to include details of the
   API requests in the logs.
 
+Contributions
+---
+
+Please refer to the [NS1 Contribution Guidelines](https://github.com/ns1/community) before opening a pull request or issue.
```

Comment:
As this is the first one, maybe makes sense to nitpick and nail it down so we can basically copy-paste in other repos. Thinking it might be nicer phrased a bit more passively. maybe something like

```Pull Requests and issues are welcome. See [NS1 contribution guidelines] for more information.```